Assets/_System/Scripts/TurnBasedScripts/CombatPlayer.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class CombatPlayer
```

Comment:
Could we rename this to something more generic? Maybe something like `Unit`? A Unit can represent any character on the board. While a `CombatPlayer` only represents the player's unit.